src/PokemonGymOwner.java

```diff
@@ -0,0 +1,38 @@
+import java.util.List;
+
+public class PokemonGymOwner extends PokemonTrainer {
+
+//    8 variables (6 in the list)
+    private String name;
```

Comment:
Deze klasse heeft geen "name" en "pokemons" nodig, want die zitten al in de super. 
In de constructor geef je deze waarden ook door aan de constructor, vandaar dat ze hier op null blijven staan.